.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+python:
+  - "3.6"
+jobs:
+ include:
+   - stage: Version Updation for sdk config
+     language: python
+     if: tag IS present
+     script:
+      - git clone https://pallavigopi:$BITBUCKET_TOKEN@bitbucket.org/shoonyacloud/esper-sdk-config.git
```

Comment:
your personal account?

Reply:
@pallavigopi  make user also env variable...we can replace later with the programmatic user.